src/Content/RelationContent.php

```diff
@@ -427,6 +428,9 @@ protected function getStandardTable(mixed $entity): Table
         $targetDefinition->configureTable($table);
         $targetDefinition->configureTableActions($table);
         $targetDefinition->configureActions(null);
+        if ($targetDefinition instanceof AbstractDefinition) {
```

Comment:
if the method `configureTableExporter` is only defined in the `AbstractDefinition` but not in the `DefinitionInterface` you should define it there and avoid this useless check!

Reply:
Done, the configureTableExporter function is now in the `DefinitionInterface `